app/models/bibliography.rb

```diff
@@ -40,8 +40,8 @@ def render(format = 'html')
   # of cases.
   def remove_latex_markup!
     @bibliography.each do |item|
-      item.title = strip_latex(item.title)
-      item.booktitle = strip_latex(item.booktitle)
+      item.title = strip_latex(item.title) if item.respond_to?(:title)
+      item.booktitle = strip_latex(item.booktitle) if item.respond_to?(:booktitle)
```

Comment:
I'm wondering if this isn't a good opportunity for either a NullObject pattern or to not process these kinds of things in general?  I worry that we'll regularly be asking questions about the object's interface (which couples this code to tightly together).

Reply:
Good idea. I put that into a ticket -- https://github.com/sul-dlss/exhibits/issues/652